gerrit-start.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env sh
 set -e
-#TODO:Not sure if gerrit can be stopped properly...
```

Comment:
This line was added from the beginning. It do looks confusing right now. I think it the time to remove it unless there are some evidences show that the container can't be stopped properly.